manifests/40-rbac.yaml

```diff
@@ -75,7 +75,7 @@ rules:
   verbs: ["update"]
 # Needed by the core operator functionality.
 - apiGroups: ["machineconfiguration.openshift.io"]
-  resources: ["kubeletconfigs", "machineconfigs"]
+  resources: ["kubeletconfigs", "machineconfigs", "containerruntimeconfigs"]
```

Comment:
for the definitive fix, this needs to be on its own entry because we will totally need to list, get and watch `containerruntimeconfigs` objects, but we will not need to create, delete, update

Reply:
Done. Thanks!